### PR TITLE
target netstandard 1.6

### PR DESCRIPTION
**Pull request by drewfreyling**
How do you want the build process to work? I'm assuming we will need .net core SDK now.

### CODE REVIEW

**Comment by drewfreyling** [comment]
Fixes #26

**Comment by robert-ovens** [comment]
Hi @drewfreyling and @danbarua. Regarding the build process we could use circleci and run the build in a couple of containers. @danbarua where are you currently running builds?

**Comment by danbarua** [comment]
AppVeyor right now but open to change if it works for you.

File: SharedAssemblyInfo.cs [hunk 1]
```diff
@@ -5,4 +5,4 @@
 [assembly: AssemblyCompany("Dan Barua, Matt Vane, Thomas Eizinger, Richard Bennett")]
 [assembly: AssemblyProduct("Nancy.Hal")]
 [assembly: AssemblyCopyright("Copyright (C) Dan Barua, Matt Vane, Thomas Eizinger, Richard Bennett and other contributors")]
```
Comment (danbarua) [inline SharedAssemblyInfo.cs]: Please add yourself to the Authors list :)

**Comment by danbarua** [comment]
It looks like I didn't get around to setting up CI for this project.
I'm happy to use Circle or Appveyor, whatever suits.
The build script only works on Windows so while we are looking at this we could migrate away from Rake to Cake for x-plat builds.

File: src/Nancy.Hal.Tests/JsonResponseProcessorTests.cs [hunk 2]
```diff
@@ -11,15 +11,33 @@
...
+        {
+            var environment =
+                new DefaultNancyEnvironment();
+
+            environment.Tracing(
+                enabled: true,
+                displayErrorTraces: true);
+
+            environment.Json();
+            environment.Globalization(new[] { "en-US" });
```
Comment (danbarua) [inline src/Nancy.Hal.Tests/JsonResponseProcessorTests.cs]: Is this required to be set? (Did we introduce a breaking change that users need to be aware of?)

Comment (drewfreyling) [reply src/Nancy.Hal.Tests/JsonResponseProcessorTests.cs]: It is for the tests to run. Nancy introduced the breaking change in v2 (see https://github.com/NancyFx/Nancy/commit/793104fcc7468cacd90c5bdd4077789fd1f04cd0) that requires a NancyEnvrionment to be added to the context. If you don't set up json, globalization and tracing you will get null ref issues running the tests.

**Comment by danbarua** [comment]
Is this PR obsolete now?

**Pull request closed without merging**